### ISSUE TITLE
openresty/openresty:alpine not using latest (master) version

### ISSUE THREAD

**Issue by EricTendian** [openresty/docker-openresty#176]
It appears that yesterday (3/25/2021) the Docker image for Alpine was tagged with an older version, which led to some downstream Docker images breaking.

What I ran:

```bash
$ docker pull openresty/openresty:alpine
alpine: Pulling from openresty/openresty
e95f33c60a64: Already exists 
527c4c56d8ce: Already exists 
b9ca6d68a283: Already exists 
c8b5785c6769: Already exists 
Digest: sha256:c3fc238bf4c448dd1c0751f51a1a267e4d77179abbfd573e1d570e06e0da0e46
Status: Downloaded newer image for openresty/openresty:alpine
docker.io/openresty/openresty:alpine
$ docker inspect openresty/openresty:alpine | jq '.[].Config.Labels'
{
  "maintainer": "Evan Wies <evan@neomantra.net>",
  "resty_add_package_builddeps": "",
  "resty_add_package_rundeps": "",
  "resty_config_deps": "--with-pcre     --with-cc-opt='-DNGX_LUA_ABORT_AT_PANIC -I/usr/local/openresty/pcre/include -I/usr/local/openresty/openssl/include'     --with-ld-opt='-L/usr/local/openresty/pcre/lib -L/usr/local/openresty/openssl/lib -Wl,-rpath,/usr/local/openresty/pcre/lib:/usr/local/openresty/openssl/lib'     ",
  "resty_config_options": "    --with-compat     --with-file-aio     --with-http_addition_module     --with-http_auth_request_module     --with-http_dav_module     --with-http_flv_module     --with-http_geoip_module=dynamic     --with-http_gunzip_module     --with-http_gzip_static_module     --with-http_image_filter_module=dynamic     --with-http_mp4_module     --with-http_random_index_module     --with-http_realip_module     --with-http_secure_link_module     --with-http_slice_module     --with-http_ssl_module     --with-http_stub_status_module     --with-http_sub_module     --with-http_v2_module     --with-http_xslt_module=dynamic     --with-ipv6     --with-mail     --with-mail_ssl_module     --with-md5-asm     --with-pcre-jit     --with-sha1-asm     --with-stream     --with-stream_ssl_module     --with-threads     ",
  "resty_config_options_more": "",
  "resty_eval_post_make": "",
  "resty_eval_pre_configure": "",
  "resty_image_base": "alpine",
  "resty_image_tag": "3.11",
  "resty_openssl_version": "1.1.1c",
  "resty_pcre_version": "8.43",
  "resty_version": "1.15.8.3"
}
```

As you can see, the Alpine image is 3.11 and the Openresty version is 1.15.8.3, which doesn't match with the build args defined on https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile. This seems to be the case as well with the `:alpine-fat` tag.

Can the alpine tag be updated to point to the latest build again? I noticed [Travis CI](https://travis-ci.org/github/openresty/docker-openresty) hasn't had a build in a while, which may be related.

**Comment by neomantra** [comment]
Thanks for pointing that out -- I think it's from transitioning to[ travis-ci.org to travis-ci.com](https://github.com/openresty/docker-openresty/issues/169).   I didn't realize it was building from my stale [personal GH repo](https://github.com/neomantra/docker-openresty/) rather than this OpenResty GH repo.

[And I've been pushing releases for new OpenSSL versions, seeing the latest was green, and not realizing that it simply wasn't processing my pushes at all.]

I'm working on this migration today and hopefully will have it resolved soon.

I think that only the untagged (latest) is broken?  If you use a tagged version, those should be OK.

**Comment by EricTendian** [comment]
@neomantra thanks! I switched my images to a specific version and that resolved the downstream issues I was having.

**Comment by neomantra** [comment]
Thanks for seeing something / saying something!

This is fixed per #169 and both untagged and all other tags are up to date.

**Comment by EricTendian** [comment]
@neomantra thanks! Verified on my end the image is now using the correct version.
